packages/forms/src/components/checkbox/Checkbox.vue

```diff
@@ -40,6 +42,10 @@ export default defineComponent({
   props: {
     name: String,
     value: Boolean,
+    disabled: {
```

Comment:
if we are moving `disabled` into a prop, I think we need to explicitly set the attr on the elements to maintain the native HTML functionality.